docs/generated/go.md

```diff
@@ -0,0 +1,201 @@
+---
+bodyclass: docs
+headline: Go Generated Code Guide
+layout: docs
+sidenav: doc-side-quickstart-nav.html
+type: markdown
+---
+
+<p class="lead">This guide gives an overview of code generated with the protoc grpc go plugin.</p>
+
+<div id="toc"></div>
+
+## Compiler Invocation and Results of Code Generation
+This section provides instructions for using the `grpc` plugin to `protoc-gen-go`, when compiling `.proto` files with `protoc`.
+See https://developers.google.com/protocol-buffers/docs/reference/go-generated for detailed instructions on 
+how to use the `protoc` compiler the `protoc-gen-go` plugin.
```

Comment:
Also add (after removing section below): 

You can find out how to define a gRPC service in a .proto file in [Service Definitions](../guides/concepts.html#service-definition).